### ISSUE TITLE
Mark repo as unsupported?

### ISSUE THREAD

**Issue by lathonez** [driftyco/ionic-unit-testing-example#23]
Hey,

Lots of people from your community are putting their time into improving this example.

https://github.com/driftyco/ionic-unit-testing-example/issues/6#issuecomment-29115245

If the above comment is accurate, do you think it'd be worth stating on the readme that there's no point raising issues or PR (or perhaps disable that altogether if that's possible)?

If the above comment is not accurate - great stuff 😸 

Let us know

**Comment by abierbaum** [comment]
Would be great to have a supported repository where the Ionic dev team and the community could work together to document best practices and methods for unit testing in Ionic.  If this repo isn't it, then we need to find out how to get one together.

**Comment by KyleMcNutt** [comment]
I for one think that the Ionic dev team should absolutely support an example project in order to display best practices for how to do testing (both unit and e2e) in Ionic. This was clearly the intention when starting this repo, and if it turns out not to be then the community should rally around one, similar to what has already happened in @lathonez's project.

**Comment by lathonez** [comment]
@KyleMcNutt agreed. I was hoping to deprecate mine for an official version. I don't really mind if it's baked into the ionic-app-scripts or a standalone repo. Just somewhere we can get things working and have a discussion.

**Comment by mlynch** [comment]
Just added @leifwells as a maintainer

**Comment by leifwells** [comment]
I'm here at [ng-conf](https://www.ng-conf.org/), so don't expect a lot of interaction from me before Saturday (April 8).

Contact me via gmail at leif.wells if you have thoughts.

**Comment by lathonez** [comment]
Thanks for the update Max, great to have @leifwells on.

**Comment by leifwells** [comment]
@lathonez If you have the time, could you do a review of any of the PRs we have in the queue? Make comments in code where you have questions/requests or comment on the main thread if you think it is ready for merge.

Anyone else can do the same. Review this code like people you respect will be using it. Set your standards high. Keep your comments civil. This project is by the community and for the community.

**Comment by lathonez** [comment]
> There's certainly room for alternative testing projects with a lot more bells and whistles that we could link to in addition to this.

I've decided to maintain clicker for the time being. I have a ~community over there who are bit stuck (as I am myself) since I looked into deprecation.

Also as the team are going to be baking this into `ion-app-scripts` at some point, it doesn't make a lot of sense for us to migrate existing code to this setup if there is an alternative - we'll just end up migrating again (to `ion-app-scripts`) down the line.

Am not interested in diluting the community on here - will reference and recommend this project on my blog and clicker readme.

**Comment by mlynch** [comment]
@lathonez more options is always better. We want to keep a simple unit testing setup without a ton of features so we can keep that core stable, but there's so many ways to go here beyond that that the community should be able to offer compelling value on top
